sematic/container_images.py

```diff
@@ -35,27 +34,21 @@ def get_image_uris() -> Dict[str, str]:
     -------
     The URI of the image to be used in this execution.
     """
+
```

Comment:
Can you add to the docstring an explanation of the expected format of the `SEMATIC_CONTAINER_IMAGE_URIS` env var, along with an example of what it could look like?

Reply:
👍